tests/tests/common_update.py

```diff
@@ -103,8 +105,8 @@ def update_image_successful(host_ip,
         Logs will not be retrieved, and result in 404.
     """
 
-    previous_inactive_part = Helpers.get_passive_partition()
-    with Helpers.RebootDetector(host_ip) as reboot:
+    previous_inactive_part = Helpers.get_passive_partition(device)
```

Comment:
This is what I am talking about below, with regards to maybe embedding the helper in the `MenderDevice`

Reply:
Got it :+1: